recipes/ipyfilechooser/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "ipyfilechooser" %}
+{% set version = "0.3.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 705a8a0d36b87414a72ab81a0327d3201a85e8f68ba1c70aa4f0250e1192a0db
+
+build:
+  number: 0
```

Comment:
This package can likely be `noarch: python`. See:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
I'll update the PR.

Reply:
`noarch: python` was added and the PR updated.